libshaderc/src/shaderc_private.h

```diff
@@ -42,6 +43,46 @@ struct shaderc_compilation_result {
       shaderc_compilation_status_null_result_object;
 };
 
+// Compilation result class using a vector for holding the compilation
+// output data.
+class shaderc_compilation_result_vector : public shaderc_compilation_result {
```

Comment:
At this point, we should seriously consider having a third subclass that contains a text result.  It's weird to have two different compilation_results for SPIR-V words but only one compilation_result for two different outputs (text vs. SPIR-V).